examples/java/src/main/java/org/apache/beam/examples/complete/TopWikipediaSessions.java

```diff
@@ -136,6 +141,20 @@ public void processElement(ProcessContext c, BoundedWindow window) {
     }
   }
 
+  static class ParseTableRowJson implements SerializableFunction<String, TableRow> {
+    private final ObjectMapper mapper =
+        new ObjectMapper().disable(SerializationFeature.FAIL_ON_EMPTY_BEANS);
```

Comment:
I actually just copied what was done here before using `TableRowJsonCoder`
If there is a different API you would rather this example would use, I'll make the change.
I'm assuming you meant using this method? https://github.com/apache/beam/blob/master/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java#L3128